### ISSUE TITLE
malloc_consolidate(): invalid chunk size

### ISSUE THREAD

**Issue by sqhendr** [Sigil-Ebook/Sigil#636]
I'm using Sigil 1.7 on Ubuntu 20.04 and I'm suddenly getting the following error:

malloc_consolidate(): invalid chunk size

followed by a core dump

I'm happy to provide any information I can to see if this can be resolved.

**Comment by dougmassay** [comment]
Steps to consistently reproduce would be a great start.

**Comment by sqhendr** [comment]
I open sigil in the terminal, Sigil starts up and I see the entire interface. Then I get the following:

Debug: regenerating tag list
Debug: regenerating tag list
Debug: regenerating tag list
Debug: regenerating tag list
malloc_consolidate(): invalid chunk size
Aborted (core dumped)

and the gui is gone.

**Comment by sqhendr** [comment]
Sometimes it allows me to put in one character before the core dump

**Comment by sqhendr** [comment]
I did a little system cleaning and tried it again. It went for a bit longer, and then after a long series of Debug: regenerating tag list:

nouveau: kernel rejected pushbuf: No such file or directory
nouveau: ch3: krec 0 pushes 0 bufs 2 relocs 0
nouveau: ch3: buf 00000000 00000002 00000004 00000004 00000000
nouveau: ch3: buf 00000001 00000006 00000004 00000000 00000004
Segmentation fault (core dumped)

**Comment by dougmassay** [comment]
That's likely a problem between your graphics driver and your system's QtWebEngine. From a terminal try:

QTWEBENGINE_CHROMIUM_FLAGS="--disable-gpu" sigil

If Sigil works, then the system nouveau video driver is insufficient to support QtWebEngine's use of hardware acceleration.

**Comment by dougmassay** [comment]
Nouveau (depending on the version) might not be able to keep up with QtWebEngine.

**Comment by sqhendr** [comment]
That makes sense. I was doing a little research on nouveau myself, because I'm normally using a proprietary NVIDIA driver. Looks like a recent Ubuntu update may have gotten things bunged up a little. I did some reinstalling of packages and now that the NVIDIA driver appears to be functioning again, I've been able to work in Sigil for a good while with no crashing. 

Sorry about that. My system was the problem. I do appreciate you helping me get on the right path for a solution. I'm a huge fan of Sigil and that got me sweating a bit.
